### ISSUE TITLE
python3 compatability 

### ISSUE THREAD

**Issue by Ekultek** [Ekultek/Zeus-Scanner#7]
Alright so there's a couple issues:

```
root@baal-Aspire-5733Z:~/bin/python/zeus-scanner# python3 zeus.py
Traceback (most recent call last):
  File "zeus.py", line 8, in <module>
    import httplib as http_client
ImportError: No module named 'httplib'
```

this is an easy fix just catch the `ImportError` and `import http.client as http_client`. See here: https://stackoverflow.com/questions/13778252/import-httplib-importerror-no-module-named-httplib

Once that is done you'll get this error:

```
root@baal-Aspire-5733Z:~/bin/python/zeus-scanner# python3 zeus.py
Traceback (most recent call last):
  File "zeus.py", line 13, in <module>
    from var import blackwidow
  File "/home/baal/bin/python/zeus-scanner/var/blackwidow/__init__.py", line 5, in <module>
    import lib.settings
  File "/home/baal/bin/python/zeus-scanner/lib/settings.py", line 12, in <module>
    import whichcraft
ImportError: No module named 'whichcraft'
```

Good luck with that one, because I have no clue how to get around it.

**Comment by cclauss** [comment]
python3 -m pip install -r requirements.txt

**Comment by Ekultek** [comment]
```
Traceback (most recent call last):
  File "zeus.py", line 16, in <module>
    from lib.attacks.admin_panel_finder import main
  File "/home/baal/bin/python/zeus-scanner/lib/attacks/admin_panel_finder/__init__.py", line 2, in <module>
    import urllib2
ImportError: No module named 'urllib2'
```

**Comment by Ekultek** [comment]
from what I found `urllib2` is not just `urllib`

**Comment by cclauss** [comment]
http://python-future.org/compatible_idioms.html is the cheatsheet...  I will come back with a more complete PR

**Comment by Ekultek** [comment]
So the nmap not found in PATH means that nmap isn’t in your env path, move it, or a copy of it, to /usr/bin Zeus will try to automatically find it in your system, but the searching can take awhile so it’s easier just to have it in your PATH

**Comment by Ekultek** [comment]
I found some info on people continually having issues with this, seems to be a flaw in the URL this might be a pain in the ass to fix on everyone’s system https://stackoverflow.com/questions/21516883/python-nmap-portscanner-error/31563964#31563964

**Comment by Ekultek** [comment]
Most support done here https://github.com/Ekultek/Zeus-Scanner/commit/f38ec5f65fa510fc7426769e95028d85b03b84f2

**Comment by Ekultek** [comment]
```
Traceback (most recent call last):
  File "zeus.py", line 401, in <module>
    agent=agent_to_use
  File "/home/baal/bin/python/zeus-scanner/var/google_search/search.py", line 265, in parse_search_results
    retval.add(url.split("&amp;")[0])
TypeError: a bytes-like object is required, not 'str'
[08:31:27 INFO] Zeus got an unexpected error and will automatically create an issue for this error, please wait...
[08:31:27 INFO] getting authorization...
[08:31:27 ERROR] ran into exception ''str' object has no attribute 'decode'' exception has been saved to log file...
Traceback (most recent call last):
  File "zeus.py", line 401, in <module>
    agent=agent_to_use
  File "/home/baal/bin/python/zeus-scanner/var/google_search/search.py", line 265, in parse_search_results
    retval.add(url.split("&amp;")[0])
TypeError: a bytes-like object is required, not 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "zeus.py", line 408, in <module>
    request_issue_creation()
  File "/home/baal/bin/python/zeus-scanner/var/auto_issue/github.py", line 65, in request_issue_creation
    token = decode(n, encoded)
  File "/home/baal/bin/python/zeus-scanner/var/auto_issue/github.py", line 31, in decode
    token = token.decode("base64")
AttributeError: 'str' object has no attribute 'decode'
[08:31:27 INFO] Zeus got an unexpected error and will automatically create an issue for this error, please wait...
[08:31:27 INFO] getting authorization...
Traceback (most recent call last):
  File "zeus.py", line 401, in <module>
    agent=agent_to_use
  File "/home/baal/bin/python/zeus-scanner/var/google_search/search.py", line 265, in parse_search_results
    retval.add(url.split("&amp;")[0])
TypeError: a bytes-like object is required, not 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "zeus.py", line 408, in <module>
    request_issue_creation()
  File "/home/baal/bin/python/zeus-scanner/var/auto_issue/github.py", line 65, in request_issue_creation
    token = decode(n, encoded)
  File "/home/baal/bin/python/zeus-scanner/var/auto_issue/github.py", line 31, in decode
    token = token.decode("base64")
AttributeError: 'str' object has no attribute 'decode'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "zeus.py", line 562, in <module>
    request_issue_creation()
  File "/home/baal/bin/python/zeus-scanner/var/auto_issue/github.py", line 65, in request_issue_creation
    token = decode(n, encoded)
  File "/home/baal/bin/python/zeus-scanner/var/auto_issue/github.py", line 31, in decode
    token = token.decode("base64")
AttributeError: 'str' object has no attribute 'decode'
```

**Comment by Ekultek** [comment]
```
Traceback (most recent call last):
  File "zeus.py", line 401, in <module>
    agent=agent_to_use
  File "/home/baal/bin/python/zeus-scanner/var/google_search/search.py", line 265, in parse_search_results
    retval.add(url.split("&amp;")[0])
TypeError: a bytes-like object is required, not 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/baal/bin/python/zeus-scanner/var/auto_issue/github.py", line 86, in request_issue_creation
    req = urllib2.Request(
AttributeError: module 'urllib' has no attribute 'Request'
```

**Comment by Ekultek** [comment]
```
Traceback (most recent call last):
  File "zeus.py", line 401, in <module>
    agent=agent_to_use
  File "/home/baal/bin/python/zeus-scanner/var/google_search/search.py", line 265, in parse_search_results
    retval.add(url.split("&amp;")[0])
TypeError: a bytes-like object is required, not 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/baal/bin/python/zeus-scanner/var/auto_issue/github.py", line 90, in request_issue_creation
    urllib2.urlopen(req, timeout=10).read()
  File "/usr/lib/python3.5/urllib/request.py", line 163, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.5/urllib/request.py", line 464, in open
    req = meth(req)
  File "/usr/lib/python3.5/urllib/request.py", line 1183, in do_request_
    raise TypeError(msg)
TypeError: POST data should be bytes or an iterable of bytes. It cannot be of type str.
```

**Comment by Ekultek** [comment]
Going to put this `ON HOLD` until I've gotten all the bugs patched and fixed, feel free to create pull requests if you want to continue the development for it, I have no problems with that. I'm just not going to be focusing on this very much right now.

**Comment by Ekultek** [comment]
Not important anymore
